services/simcore/docker-compose.deploy.aws.yml

```diff
@@ -7,6 +7,10 @@ services:
       placement:
         constraints:
           - node.role == worker
+  efs-guardian:
+    volumes:
+      - efs_volume:/data/efs
```

Comment:
if you use volumes that are kept on the machine, then you service should be bound strictly to this particular machine. If this is the case, then `simcore` label does not fit since this label is attached to multiple machines. We can discuss ...

Reply:
Ok, I probably copy pasted it, should I remove it then?

Reply:
As discussed as this volume is using distributed file system, it can be reused across multiple instances.